src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -328,16 +336,16 @@ public boolean meetsCriteria(ProgrammeMembership programmeMembership,
   }
 
   /**
-   * TEMPORARY (I hope). Identifies placements that fall within the pilot group 2024.
+   * Check whether a programme membership's trainee should receive the given message-type
+   * notification.
    *
-   * @param localOffice The placement local office.
-   * @param specialty   The placement specialty.
-   * @param startDate   The placement start date.
-   * @return true if the placement is in the pilot group, otherwise false.
+   * @param programmeMembership The programme membership to check.
+   * @param messageType         The potential notification message type.
+   * @return true if the trainee should receive the notification, otherwise false.
    */
-  protected boolean isInPilot(String localOffice, String specialty, LocalDate startDate) {
-
-    // for now, say no-one is in pilot
-    return false;
+  public boolean programmeMembershipIsNotifiable(ProgrammeMembership programmeMembership,
```

Comment:
This pass-thru function is a bit random, but it avoids the InAppService from having to use the MessagingControllerService directly.